### DISCUSSION TITLE
Test case provisions: Check whether the certificate backend is reachable

### DISCUSSION THREAD

**Discussion by fue-medatixx** [Digitaler-Impfnachweis/certification-apis#77] [Q&A]
In the test case provisions it is required to check whether the certificate backend can be reached. 
Is it possible to add an api endpoint something like "echo" or "IamAlive"?
With the current APIs, we only see the possibility of sending the challenge request as a test and checking whether the http status is ok.

**Comment by thinkberg** [comment]
Try `/` and see if it returns ok and some nice hello message.

**Reply by fue-medatixx** [reply]
Which exact uri is meant by "/"  ?
https://id.impfnachweis.info/ or
https://api.impfnachweis.info/ 
or both ?
Or a special subpath?

**Reply by thinkberg** [reply]
Check the api endpoint.

**Reply by fue-medatixx** [reply]
OK thanks. It works in the RU (https://api.ru.impfnachweis.info/).
We can assume that this will be a valid check in the PU for the foreseeable future?

Result: {"version":"1.0","ok":true,"data":"Hallo, Hola, こんにちは, Hello, Salut, Hej, this is the Ubirch Certify Service."}
Maybe you can set the json version property to the current published api-version at time? E.g. 2.1.0 for RU at the moment.
